cookbooks/lib/features/mongodb_spec.rb

```diff
@@ -7,12 +7,17 @@
   describe 'mongo commands', sudo: true do
     before :all do
       sh('sudo service mongodb start')
-      tcpwait('127.0.0.1', 27_017)
+      procwait(/\bmongod\b/)
       sh('mongo --eval "db.testData.insert( { x : 6 } );"')
+      sleep 3 # HACK: thanks a bunch, Mongo
```

Comment:
This makes zero sense to me.  I did a bunch of experimentation and found that issuing `mongo --eval` commands in rapid succession would result in failures *most* of the time 😐